code/modules/holoparasite/holoparasite_manifest.dm

```diff
@@ -32,7 +32,7 @@
 			mr_hud?.begin_timer(HOLOPARASITE_MANIFEST_COOLDOWN)
 		playsound(loc, 'sound/creatures/holopara_summon.ogg', vol = 45, extrarange = HOLOPARA_MANIFEST_SOUND_EXTRARANGE, frequency = 1)
 		add_filter("holopara_manifest", 1, gauss_blur_filter(size = 4))
-		transition_filter("holopara_manifest", 1.2 SECONDS, list("size" = 0), easing = SINE_EASING, loop = FALSE)
+		transition_filter("holopara_manifest", 1.2 SECONDS, list("size" = 0), easing = SINE_EASING, loop = 0)
```

Comment:
```suggestion
		transition_filter("holopara_manifest", 1.2 SECONDS, list("size" = 0), easing = SINE_EASING, loop = FALSE)
```

Why was this changed from `FALSE` to `0`?

Reply:
Loops is the number of times you want it to loop and not a boolean. It's pedantic, but i thought I'd make it the correct type anyway

Reply:
Makes sense to me